packages/select/src/Select/Select.tsx

```diff
@@ -90,6 +111,8 @@ export const Select: React.FC<SelectProps> & SubComponentProps = ({
     }
   }, [state.isOpen])
 
+  const shouldShowValidationMessage = !isDisabled && validationMessage
```

Comment:
Oh, according to design the message should show regardless of disabled state

Reply:
I think this maybe a mistake but I will ask Elliot

Reply:
wait what message are you seeing in the design?

The description message will show regardless the validation won't show if disabled

Reply:
![image](https://user-images.githubusercontent.com/820635/207771060-c584da64-28dd-4d85-85f8-7bd54e323965.png)

I assume this condition will prevent the 4th column from happening?

Reply:
This is something @HeartSquared and I were just talking about. The date picker has this opinion built in, which I'm not a fan of. I reckon we should leave this up to consumers.

Reply:
This is a strange thing tbh. If it's disabled, does it even make sense to have any validation message on it? I feel like I remember having a chat with Jason about it earlier when we were working on the DatePicker and we figured it made no sense so we put the conditions in.
But yeah, I guess maybe we should just let them do it; weird as it is. Perhaps worth revisiting the whole validation structure pattern in general.